backend/lib/dbHelpers.js

```diff
@@ -38,8 +43,9 @@ module.exports = (client) => {
       cb(null, reply)
     })
   }
+  this.getProductById = Bluebird.promisify(getProductById)
 
-  this.getProductsByCategories = (categoriesArr, cb) => {
+  getProductIdsByCategories = (categoriesArr, cb) => {
     client.SINTER(...categoriesArr, (err, reply) => {
       if (err) {
         console.log(err)
```

Comment:
For lines 51, 83 and 100 wasn't this one of the anti-patterns that was covered in the node talk at FAC a few weeks ago? If there is an error, you log it, but then the code continues executing anyway as if there was no error. Is this the desired behaviour? The idea behind the pattern of having a callback that takes two arguments is that the first argument is non-null if there's an error (i.e. call `cb(err)`). This will also help you when promisifying these methods, as a callback with a non-null first argument will result in the promise being rejected. This can be caught at the top level by appending a `.catch(function() {...})` to your then-chain. That would be closer to proper error-handling.